.dockerignore

```diff
@@ -0,0 +1,3 @@
+/data_resized
```

Comment:
You can take a few more things from https://github.com/marco-c/autowebcompat/blob/master/.gitignore (specific to this project) and https://github.com/mozilla/bugbug/blob/master/.dockerignore (generic stuff).